node/pod.go

```diff
@@ -344,7 +345,11 @@ func (pc *PodController) deletePodHandler(ctx context.Context, key string) (retE
 	ctx, span := trace.StartSpan(ctx, "processDeletionReconcilationWorkItem")
 	defer span.End()
 
-	namespace, name, err := cache.SplitMetaNamespaceKey(key)
+	idx := strings.LastIndex(key, "-")
```

Comment:
I do not recommend we change out the key format. That seems like it could quickly, and easily generate confusion.

Reply:
have changed "-" to "/"